docker-config/es/Dockerfile

```diff
@@ -1,4 +0,0 @@
-FROM elasticsearch:1.5
```

Comment:
We should keep this as a reference to the image we're using in the dev deployment.